catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/image/NitfPreStoragePlugin.java

```diff
@@ -208,7 +208,8 @@ private ContentItem createDerivedImage(String id, String qualifier, BufferedImag
     }
 
     private String buildDerivedImageTitle(String title, String qualifier) {
-        String rootFileName = FilenameUtils.getBaseName(title);
+        // filter non-word characters from the filename for file system storage
+        String rootFileName = FilenameUtils.getBaseName(title).replaceAll("[\\W]", "");
```

Comment:
If all the characters in the `ftitle` where special characters, you'd end up with a filename of `{qualifier}-.jpg`. You should probably handle the edge case where that happens.

Should also add unit tests to check for the various names generated, though as this is a private method it will be a little painful to test.